docs/pages/includes/enterprise/ent-user-prereqs.mdx

```diff
@@ -0,0 +1,30 @@
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted">
+- Installed Enterprise version of Teleport (downloaded via the [Enterprise dashboard](https://dashboard.gravitational.com/web/login)).
+- Maintain via the `tctl` enterprise admin tool via the Desktop (recommended) or have administrative access on the Teleport auth service machine.
```

Comment:
I think we can condense this and the following bullets into a single bullet, since they both instruct you to download `tctl`. Also, I think we can make it a lot simpler by borrowing from the current version of `edition-prereqs-tabs.mdx` (this example also includes `tsh`, which I think we should mention if we're going to show `tsh` commands):

```md
- The `tctl` admin tool and `tsh` client tool version >= 
(=teleport.version=), installed on your local machine.

You can download these by visiting the [customer portal](https://dashboard.gravitational.com/web/login).
```

We can then change `tctlconnection.mdx` to make it explicit that you should run the `tctl` command on your local machine.